src/lib/expansion/accordion.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  Directive,
+  ElementRef,
+  EventEmitter,
+  Host,
+  Input,
+  Output,
+  ViewEncapsulation,
+  AfterContentInit,
+  OnDestroy,
+  ContentChildren,
+  QueryList,
+  Optional,
+  forwardRef,
+} from '@angular/core';
+import {
+  trigger,
+  state,
+  style,
+  transition,
+  animate,
+} from '@angular/animations';
+import {coerceBooleanProperty} from '../core/coercion/boolean-property';
+import {Subscription} from 'rxjs/Subscription';
+import {Subject} from 'rxjs/Subject';
+import 'rxjs/add/operator/takeUntil';
+
+/** Used to generate unique ID for each expansion panel. */
+let nextId = 0;
+
+export class CdkAccordionChild implements OnDestroy {
+  /** Event emitted every time the MdAccordianChild is closed. */
+  @Output() close = new EventEmitter<null>();
+  /** Event emitted every time the MdAccordianChild is opened. */
+  @Output() open = new EventEmitter<null>();
+  /** Event emitted when the MdAccordianChild is destroyed. */
+  @Output() destroy = new EventEmitter<null>();
+  /** Whether the MdAccordianChild is expanded. */
+  expanded: boolean;
```

Comment:
Since this isn't an `@Input`, we have control over this being a `boolean` here correct?

In the abstract, it would just be a `boolean`, and then in the implementation (`MdExpansionPanel`) it would use `coerceBooleanProperty`.